scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -51,6 +51,20 @@ trait Config extends Serializable {
     }
 
   /**
+   * Add files to be localized to the config. Intended to be used by user code.
+   * @param cachedFiles CachedFiles to be added
+   * @return new Config with cached files
+   */
+  def addDistributedCacheFiles(cachedFiles: CachedFile*): Config =
```

Comment:
@benpence I have 2 concerns about API like that:
1. DistributedCacheFile is the property of config, not an execution.
2. at now we don't have the ability to pass any changes in config to next operations, so we should create distributed cache before execution, then even if we will have that ability it's will not compatible with older code (you can not use same code for Job API and Execution API), also it's little bit ugly to use for multiple files.